digdag-core/src/main/java/io/digdag/core/schedule/SchedulerManager.java

```diff
@@ -82,10 +94,47 @@ private Scheduler getScheduler(Config schedulerConfig, ZoneId workflowTimeZone)
             c.set("_command", command);
         }
 
+        for(String param : BUILT_IN_SCHEDULE_PARAMS){
+            if (param.equals("skip_on_overtime"))
```

Comment:
You don't need these `if` statements. You can use following instead:
```suggestion
            usedKeys.add(param);
```

Reply:
Thanks, I fixed to use that.
https://github.com/treasure-data/digdag/pull/1381/commits/e9740105d2abb8eae39c1bab699ee0816fdf4de8